pom.xml

```diff
@@ -136,7 +136,7 @@
         <cuda.javacpp.version>${cuda.version}-${javacpp.version}</cuda.javacpp.version>
         <nexus-staging-maven-plugin.version>1.6.8</nexus-staging-maven-plugin.version>
         <reactive-streams.version>1.0.3</reactive-streams.version>
-
+        <javacpp.platform>windows-x86_64</javacpp.platform>
```

Comment:
This is not how we should do this. This property should be given from outside the jvm as a system property. This should be taken out.

Reply:
Yes Sham we understood the same it might be mistakenly committed during our testing .will remove and commit.